src/styles/MuiThemeProvider.d.ts

```diff
@@ -3,7 +3,7 @@ import { Theme } from './createMuiTheme';
 
 export interface MuiThemeProviderProps {
   theme: Theme | ((outer: Theme | null) => Theme);
-  sheetsManager?: Object;
+  sheetsManager?: Map<any, any>;
```

Comment:
Based on the JavaScript, it looks like the most accurate type would be

```ts
sheetsManager?: SheetsManager;
```

with something like the following:

```ts
import { StyleSheet } from ‘jss’;

type SheetsManager = Map<StylesCreator, Map<Theme, SheetManagerTheme>>;

interface StylesCreator {
  create(theme: Theme, name: string): StyleRules;
  options: { index: number };
  themingEnabled: boolean;
}

interface SheetManagerTheme {
  refs: number;
  sheet: StyleSheet<string>;
}
```

(requires adding a dependency on `@types/jss`).

Reply:
I added @types/jss and update the generic type. If we have no problem with the extra dependency then I think it should be ok.